wire/message.go

```diff
@@ -22,9 +22,28 @@ const MessageHeaderSize = 24
 // header.  Shorter commands must be zero padded.
 const CommandSize = 12
 
-// MaxMessagePayload is the maximum bytes a message can be regardless of other
-// individual limits imposed by messages themselves.
-var MaxMessagePayload = uint32(1024*1024*32) * 2 // 32MB
+// ebs is the excessive block size, used to determine reasonable maximum message sizes.
+var ebs uint32
+
+// SetLimits initializes various message limits which change depending on the maximum
+// block size we will deal with.
+func SetLimits(excessiveBlockSize uint32) {
+	ebs = excessiveBlockSize
+}
+
+func getEBS() uint32 {
```

Comment:
We should export this and just call it `EBS()`. I want to add support for the UserAgent and will need that in `peer.go` =)

Reply:
Ok. Also if we will use a default as above, then I will just remove this getter.

Reply:
Yep that works.

Reply:
removed the getter